interactem-cli/pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
```

Comment:
for packaging, I think we should 

1. Follow the typer guide here: https://typer.tiangolo.com/tutorial/package/
2. put this into a directory called "cli" instead of "interactem-cli"